### PR TITLE
search page layout + translations

### PR DESCRIPTION
**Pull request by cathydeng**
- reorganizing translation files a bit
- adding translation keys to home page and search page
- stub out mobile vs desktop layout on search page

https://user-images.githubusercontent.com/1406537/175181905-108703b8-7820-4514-a5b4-493caea43fdc.mov

### CODE REVIEW

File: src/translations/en/pages.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "home": {
+    "heading": "Find behavioral health care that's right for you.",
+    "zipPrompt": "Enter your zip code to find care near your area"
+  },
+  "search": {
+    "heading": "Results near",
+    "resultCount_one": "Result",
+    "resultCount_other": "Results"
+  },
+  "commonLabels": {
```
Comment (cathydeng) [inline src/translations/en/pages.json:15]: kinda awkward naming, but dumping things here that are likely to show up on multiple pages

File: src/pages/Search.tsx [hunk 2]
```diff
@@ -42,23 +44,49 @@ function Search() {
...
+              <>
+                <h2>
+                  {searchResult.results.length}{" "}
+                  {t("pages.search.resultCount", {
+                    count: searchResult.results.length,
+                  })}
+                </h2>
+                <Grid
+                  row
+                  className="display-none tablet:display-block border-top border-base-lighter"
```
Comment (cathydeng) [inline src/pages/Search.tsx:66]: having diff things that show/hide on diff sizes felt like the most straightforward way of implementing this layout, but not sure about the accessibility implications of having results in the dom twice?

File: src/pages/Search.tsx [hunk 3]
```diff
@@ -38,27 +45,88 @@ function Search() {
     }
   }, []);
 
+  const desktopList = (results: CareProviderSearchResult[]) => (
```
[the remark this answers is not in the document]
Comment (jlhogan) [reply src/pages/Search.tsx:48]: i think my vote would be mini func components: 
```
const DesktopList = ({ results } : { results: CareProviderSearchResult[] }) => (<>...</>)
const DesktopMap = () => <>...</>
```

File: src/pages/Search.tsx [hunk 4]
```diff
@@ -38,27 +83,71 @@ function Search() {
     }
   }, []);
 
+  const mobileViewToggle = (
```
Comment (jlhogan) [inline src/pages/Search.tsx:86]: nit: make this match other mini components ?

**Review by jlhogan — APPROVED** [review]

File: src/pages/Search.tsx (hunk 4 above)
Comment (cathydeng) [reply src/pages/Search.tsx:86]: for this one i thought it was a lil easier to read in here instead of passing the state stuff around

**Pull request merged**